apps/communications/contacts/test/marionette/ice_test.js

```diff
@@ -262,28 +255,77 @@ marionette('Contacts > ICE contacts', function() {
       var save = client.findElement(selectors.formSave);
       save.enabled();
       save.click();
-    });
+    }
 
-    test('Confirm window appears with correct message', function() {
-      var confirmBody = client.helper.waitForElement(selectors.confirmBody);
+    function closeSettings() {
+      var close = client.helper.waitForElement(selectors.settingsClose);
+      subject.clickOn(close);
+    }
 
-      var expectedResult = subject.l10n(
-        '/locales-obj/en-US.json',
-        'ICEContactDelTelAll');
+    suite('With one phone number', function() {
+      setup(function() {
+        addTestContact(true);
+        setFirstContactAsICE();
```

Comment:
This should be part of the test and not the setup, as we are on a suite of contacts edition, not contacts removal